stdpopsim/catalog/arabidopsis_thaliana.py

```diff
@@ -36,7 +36,17 @@
         mutation_rate=7e-9,
         recombination_rate=8.1e-9))
 
-_genome = stdpopsim.Genome(chromosomes=_chromosomes)
+_SwarbreckEtAl = stdpopsim.Citation(
+    doi="https://doi.org/10.1093/nar/gkm965",
+    year="2007",
+    author="Swarbreck et al.",
+    reasons={stdpopsim.CiteReason.ASSEMBLY}
+)
+
+_genome = stdpopsim.Genome(
+        chromosomes=_chromosomes,
+        assembly_citations=[
+            _SwarbreckEtAl.because(stdpopsim.CiteReason.ASSEMBLY)])
```

Comment:
If the citation has `reasons` specified, then you don't need the `because()` call later. You could probably just put the ASSEMBLY citation inline here (with `reasons` given), instead of storing it in _SwarbreckEtAl, as it probably wont be used elsewhere.

Reply:
can do